lib/screenplay_web/auth_manager/auth_manager.ex

```diff
@@ -3,9 +3,10 @@ defmodule ScreenplayWeb.AuthManager do
 
   use Guardian, otp_app: :screenplay
 
-  @type access_level :: :none | :read_only | :admin
+  @type access_level :: :none | :read_only | :admin | :configurer
 
-  @screenplay_admin_group "screenplay"
+  @screenplay_admin_group "screenplay-emergency-admin"
+  @screenplay_configurer_group "screenplay-screen-configurer"
```

Comment:
Ohhhh! I just realized, we didn't need to add a new group but could just use the existing group called `screens-admin`. I'm not sure `screenplay-screen-configurer` is clearer than the name of the existing group. What do you think about nixing that new group and keeping the old one, especially since you're essentially renaming the group here to `screenplay_configurer_group` (or maybe `screens_admin_group`)

Reply:
When you refer to 'group description' are you just referring the name of the group (e.g. `screenplay-emergency-admin` or something else? If it's just the name, I feel like `screenplay-emergency-admin` does a pretty decent job of describing a group that represents users who have access to the OFM takeover tool in Screenplay, but not sure if the 'description' of a group is a specific attribute that belongs to the cognito group?

And yeah that's a good point and agreed, we should just re-use the old `screens-admin` group rather than setting up a new one for Screenplay that will contain the same users.

Reply:
[Group description](https://github.com/mbta/devops/pull/1344/files#diff-7b658ee617733625d8c755ac321b1462018a45c08a6e24f1e7b63533dae7dac7R92). I agree the name `screenplay-emergency-admin` is better than `screenplay`, but the description of "Access to the Screenplay tool" is no longer quite accurate

Reply:
Ah, that description. My gut tells me that changing the description wouldn't do anything to change what users belong to that group, but I don't know for sure. I can ask infra about that one to be sure though.